ewc-rfc005-issue-legal-person-identification-data.md

```diff
@@ -31,34 +31,42 @@ The LPID issuance process is implemented by PID providers according to national
 
 4. **Status:** The status of the legal person must be verified. These states are not yet regulated, but at the very least, the legal person for which an LPID is requested must be registered at a business register.
 
-The issuance process can either start in:
-1. a wallet application, the grahical user interface of the wallet *or*
+The issuance process can be done in:
+1. a wallet application, the grahical user interface of the wallet
 2. the eService of the PID provider.
+3. or a combination of the two above
 
 The first approach requires the wallet provider to implement support for a lookup of all PID Providers. It also requires the User (natural person or legal person, as defined in eIDAS2 [8]) to have a wallet solution [8] that includes a wallet application. Wallet applications are optional and not all legal persons may opt for a wallet solution that includes a graphical user interface. An end user representing the legal person chooses an PID Provider in the wallet application to apply for/request an LPID from. The wallet application then redirects the end user to the eService of the PID Pprovider for the issuance process. After completeing the eService procedure, the natural person is redirected back to the wallet application from where a offer reqest is initiated.
 
 The second approach has no dependency to the implementaion of the wallet or the wallet solution of the User. An end user representing the legal person goes directly to the eService of the PID Provider to apply for/request an LPID. After completeing the eService procedure, the PID Provider initiates a offer response to the wallet instance of the legal person which endpoint the representative of the lagal person supplied. The endpoint can be a url or a DID for instance. All communication between wallets is done in the backend.
 
-The picture below shows a generic outline of the LPID issuance process, the actual implemntation depends on national regulations.
-![Generic LPID issuance process](images/Generic%20LPID%20issuance%20process.png)
+The third approach combine the wallet and the E-service into one service. Where the user has a seemless experience and there is no need for the user to switch between webservices. The switching between the wallet and the e-service is done in through machine to machine in the background. 
```

Comment:
As mentioned above, the third option is covered by the original first option, but not detailed, as stated in the text, in this RFC. The second option will work with any wallet implementation, while the first option only will work with wallet solutions that include a graphical user interface offering the option to request an LPID.